force-app/main/default/lwc/donationHistory/donationHistory.js

```diff
@@ -0,0 +1,6 @@
+import { LightningElement, api } from 'lwc';
+
+export default class DonationHistory extends LightningElement {
+    @api
+    recordId;
```

Comment:
```suggestion
    @api recordId;
```

When we decorate variables, we usually do it on the same line. Decorators for functions can go above the function.